matijapetanjek LPS 181729 2

rotty3000 opened the pull request:
- LPS-181729 manually update batch enabled ResourceImpls with VulcanBatchEngineTaskItemDelegate service
- LPS-181729 export rest resource impls with service interface VulcanBatchEngineTaskItemDelegate
- LPS-181729 Refactor BatchEngineTaskItemDelegateRegistry to wait for ResourceImpls to become active
- LPS-181729 wait for PORTAL_INITIALIZED
- LPS-181729 batch-engine-service - remove the validation, batch engine executor will throw exception (will get NPE here if TaskItemDelegate is not registered in the TaskItemDelegateRegistry before BatchEngineImportTaskComponent is activated)
- LPS-181729 batch-engine-test - update test
- Revert "LPS-181729 batch-engine-service - remove the validation, batch engine executor will throw exception (will get NPE here if TaskItemDelegate is not registered in the TaskItemDelegateRegistry before BatchEngineImportTaskComponent is activated)"
- Revert "LPS-181729 batch-engine-test - update test"
- LPS-181729 always use BatchEngineUnitProcessor

Commit:
LPS-181729 export rest resource impls with service interface VulcanBatchEngineTaskItemDelegate

Commit:
LPS-181729 Refactor BatchEngineTaskItemDelegateRegistry to wait for ResourceImpls to become active

Commit:
LPS-182073 Fix visual details on new client-extension type

jkappler committed:
LPS-182317 Adds ootb utility page 500 to welcome site initializer

qz1010 committed:
LPS-182475 SF

qz1010 committed:
LPS-182475 SF

veroglez committed:
LPS-167608 Add maximum character feedback to the label of the input

veroglez committed:
LPS-167608 Add maximum characters feedback to the textarea

Commit:
LPS-181729 wait for PORTAL_INITIALIZED

dsanz committed:
LPS-177720 effectively hide the internal checkbox to the screen reader and tab order

Commit:
LPS-181729 batch-engine-service - remove the validation, batch engine executor will throw exception (will get NPE here if TaskItemDelegate is not registered in the TaskItemDelegateRegistry before BatchEngineImportTaskComponent is activated)

Commit:
LPS-167572 un-ignore since the bug was fixed

Commit:
LRQA-79256 add CanCreateVirtualInstanceViaConfig testcase

Commit:
LPS-174421 New Resource Action

Commit:
LPS-174421 Make AuditDisplayContext compatible with ResourceRequests

Commit:
LPS-174421 PortletConfigurationIcon to show export option

Commit:
LPS-174421 ResourceAction for CSV download

Commit:
LPS-181729 batch-engine-test - update test

Commit:
LPS-174421 Export all events, not just the current page

dsanz committed:
LPS-182073 build lang

Commit:
LPS-183189 Simplify permissions, restrict to admin only.

For LPS-174421.

Commit:
LPS-174421 buildLang

david-truong committed:
LPS-183473 only let recycle be emptied in production

Commit:
LPS-150114 Update macro typeAllSetupDXPCloudInfoFields

Commit:
LPS-150114 Update macro activeStatusViaDropdown

Commit:
LPS-150114 Update macro tearDownForCloudEntries

Commit:
LPS-150114 Update path e macro for the modal info

Commit:
LPS-150114 Add test AssertChangeOfCardStatusToActive

Commit:
LPS-150114 Add test AssertThatTheConfirmationIDHasBeenUpdated

Commit:
LPS-150114 Add test AssertThatTheStatusIsTheSameWhenCancelingAnAction

Commit:
LPS-150114 Auto SF

Commit:
Revert "LPS-181729 batch-engine-service - remove the validation, batch engine executor will throw exception (will get NPE here if TaskItemDelegate is not registered in the TaskItemDelegateRegistry before BatchEngineImportTaskComponent is activated)"

This reverts commit 5b1f4894104afe07c4f91c52e14c7eb64d8042cf.

Commit:
Revert "LPS-181729 batch-engine-test - update test"

This reverts commit 5e2f4ef56c08a0a1c55bf349dc10866747af9903.

david-truong committed:
LPS-183473 if item is in trash show that it is deleted in UI

Commit:
LPS-183732 Updating publication tests do not empty the recycle bin

Commit:
LPS-183732 Remove the inapplicable tests

Commit:
LPS-179282 Add paths for datasets fields tests

Commit:
LPS-179282 Add searchField macro

Commit:
LPS-179282 Create DatasetsFields.testcase file

Commit:
LPS-179282 SF

davisaints committed:
LPS-183871 Remove feature flag property

davisaints committed:
LPS-183871 Fix CanSendEmailViaActionStandalone test

Commit:
LPS-150114 Fix tests for CP2

Commit:
LPS-181729 always use BatchEngineUnitProcessor

# Conflicts:
#	modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/installer/BatchEngineFileInstaller.java

david-truong committed:
LPS-183473 don't allow empty recycle bin in quick actions when in a publication

rafaprax committed:
LPS-183802 Patch nimbus-jose-jwt

rafaprax committed:
LPS-183802 Create module to hold the outlook connector implementation

john-co committed:
LPS-183977 Create accessibility ci test suite

jaydawu committed:
COMMERCE-10603 Add and update macros

Commit:
LPS-183985 Cannot empty recycle bin within publication

Commit:
LPS-183732 Workaround the steps to view changes

jkappler committed:
LPS-184003 Convert to RFF

jaydawu committed:
COMMERCE-10603 Add test CanViewProductsWithCategoryImportedFromAnotherInstanceViaJSONAdd

ricardocousolr committed:
LPS-184014 Take possible non ROOT context into account

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

rafaprax committed:
LPS-183802 Create the MailAuthTokenProvider API to enable the implementation of specific authentication flow for each mail server provider. For now, we'll add the connector for the outlook but there are plans to add for Gmail as well

Commit:
LPS-181729 always process batch engine units in lexicographical order

Commit:
LPS-181729 register a single thread executor for batch engine

Commit:
LPS-181729 log success only when a unit was actually enqueued

Commit:
LPS-181729 compose the batch.engine.entity.class.name for Objects

Commit:
LPS-181729 register ObjectEntryResource as VulcanBatchEngineTaskItemDelegate so batch can defer processing

rafaprax committed:
LPS-183802 Baseline

thaynam committed:
LPS-183702 Add Partner Account column on the Partner Growth Deal listing

Commit:
LPS-181729 change method to pass batchEngineTaskItemDelegate

Commit:
LPS-181729 buildService

Commit:
LPS-181729 use factory to create BatchEngineTaskItemDelegate on demand

thektan committed:
LPS-176188 LPS-183987 Add delete action to Filters

thektan committed:
LPS-176188 LPS-183987 Close dropdown after deleting an item

Uses `id` if one exists so the row is re-rendered after an item is removed.

Commit:
LPS-183998 Add get export batch in macro files

Commit:
LPS-183998 Add tests on generate export endpoints for headless APIs

slnn committed:
LPS-184134 Merge all these PortletDisplayTemplateUtil into one under the API module portlet-display-template-api and apply

slnn committed:
LPS-184134 Update SF properties

slnn committed:
LPS-184134 Baseline

markocikos committed:
LPS-183656 Build lang

izaera committed:
LPS-183098 Infer template language instead of assuming velocity

markocikos committed:
LPS-183656 Implement editing of FDS Fields

BarbaraCabrera committed:
LPS-183325 Replace use of ContentDashboardGroupUtil in ContentDashboardAdminConfigurationDisplayContext

Commit:
LPS-181729 tests

kevenleone committed:
LPS-184161 Replace React Scripts by Vite

kevenleone committed:
LPS-184161 Rename filenames by .js to .jsx

izaera committed:
LPS-183098 Change argument name in Util class too

kevenleone committed:
LPS-184161 Update Client Extension entry points

rafaprax committed:
LPS-183802 Lang key

kevenleone committed:
LPS-184161 SF

aline-doleron committed:
LPS-182571 Add priority order to images

gamerson committed:
LPS-181325 fix assemble

kevenleone committed:
LPS-184161 Rename Test files extension

kevenleone committed:
LPS-184161 Add Setup Test and Cover js, jsx extension

nNilton committed:
LPS-184211 add information templates

shuyangzhou committed:
LPS-184480 Merge TransactionHandler into TransactionExecutor

shuyangzhou committed:
LPS-184480 Fix tests

Commit:
LPS-183998 Update tests by replace postAccountGroupsPageExportBatch with postAccountsPageExportBatch

dantewang committed:
LPS-184249 Scheduler: Add job to clean up stale background tasks so there's no need to do it during BackgroundTaskManagerImpl startup

dantewang committed:
LPS-184249 build lang

ces-quanhuynh committed:
LPS-183757 Center publication modal entry titles

BarbaraCabrera committed:
LPS-183325 Replace use of ContentDashboardGroupUtil in ContentDashboardAdminManagementToolbarDisplayContext

BarbaraCabrera committed:
LPS-183325 Replace use of ContentDashboardGroupUtil in ContentDashboardItemSubtypeItemSelectorView

BarbaraCabrera committed:
LPS-183325 Replace use of ContentDashboardGroupUtil in GetContentDashboardItemInfoMVCResourceCommand

BarbaraCabrera committed:
LPS-183325 Remove unused ContentDashboardGroupUtilTest

markocikos committed:
LPS-183656 Rename label

markocikos committed:
LPS-183656 Deduplicate

markocikos committed:
LPS-183656 Define cell render labels locally

Commit:
LRQA-79256 ignoring due to LPS-184310

thaynam committed:
LPS-183737 Add additional fields in Deal Reg moda

Commit:
LPS-181729 add timeout to the batch unit processing

Commit:
ISSITES-5778 apply clay variable overrides to theme css

Commit:
LPS-179282 Update teardown

Commit:
LPS-179937 Manager isn't able to comment on MDF/Claim

joelrocha68 committed:
LPS-181788 Add setup and teardown

joelrocha68 committed:
LPS-181788 Add AnalystRoleCanNotCreateEditAndDelete test

joelrocha68 committed:
LPS-181788 Add LeadRoleCanCreateEditAndDelete test

joelrocha68 committed:
LPS-181788 Add UserRoleCanNotCreateEditAndDelete test

joelrocha68 committed:
LPS-181788 SF

Commit:
LPS-183189 Follow pattern from the Instance Settings portlet to restore admin only portlet access and no assignable permissions.

For LPS-174421.

raquelpaiva committed:
LPS-184462 Breadcrumb adjustment on submit page Organization

nNilton committed:
LPS-184211 add Test to assert information templates

aline-doleron committed:
LPS-183960 Add a base URL to the requests

aline-doleron committed:
LPS-183960 Create product specifications for License Terms and Price Model

joelrocha68 committed:
LPS-181788 Fix

joelrocha68 committed:
LPS-181632 Add setup and teardown

joelrocha68 committed:
LPS-181632 Add AnalystRoleCanNotCreateEditAndDelete test

joelrocha68 committed:
LPS-181632 Add LeadRoleCanCreateEditAndDelete test

joelrocha68 committed:
LPS-181632 Add UserRoleCanNotCreateEditAndDelete test

joelrocha68 committed:
LPS-181632 SF

Commit:
LPS-181022 Feature Flag.

For LPS-174421.

joelrocha68 committed:
LPS-181632 Fix

Commit:
LRAC-13506 Add RequestToAccessDataForAllForms testcase

Commit:
LRAC-13507 Add RequestToAccessDataForSpecificForm testcase

Commit:
LRAC-13508 Add RequestToAccessDataForAllWebContent testcase

thektan committed:
LPS-176188 LPS-183987 Refactor url to separate variable

Commit:
LRAC-13509 Add RequestToAccessDataForSpecificWebContent testcase

Commit:
LRAC-13489 Add RequestToAccessDataForAllExistingPages testcase

thektan committed:
LPS-176188 LPS-183987 Add confirmation modal

thektan committed:
LPS-176188 LPS-183987 New language key

thektan committed:
LPS-176188 LPS-183987 gradlew buildLang

Commit:
LRAC-13506 Update tearDown to delete created site and users

Commit:
LRAC-13506 SF

melodywu committed:
LPS-182413 Add locators for autdit export csv

melodywu committed:
LPS-182413 Add macro for autdit csv

melodywu committed:
LPS-182413 Add testcase for autdit csv

Commit:
LPS-183998 Remove test on headless commerce admin api

Commit:
LPS-184140 Add test to edit and adapt image resolution in a publication

Commit:
LPS-184141 Add test to disable and delete image resolution in publication

dantewang committed:
LPS-184249 Remove BackgroundTaskClusterMasterTokenTransitionListener because the scheduler job can handle the cleanup when master node transition happens

Commit:
LPS-183732 Update the conflict message

david-truong committed:
LPS-184379 fix package

Hanlf committed:
LPS-180322 Update macros to include fileName and title vars

Hanlf committed:
LPS-180322 Add macros about document

Commit:
LPS-183998 Rename variable name in macro files

JavierMoral committed:
LPS-184331 Get locale from themeDisplay

Commit:
LPS-184138 Add test to add and adapt image resolution in a publication

JessieZhao0714 committed:
LPS-182445 Add testcase to check confirmation dialog of deleting application data

Commit:
LPS-167608 - Wordsmithing

Commit:
LPS-167608 - buildLang

dsanz committed:
LPS-184390 Prevents fields referencing other schemas from being available for selection

Commit:
LPS-183990 Remove setup virtual instance and tear down block from TagsFragment

Commit:
LPS-182073 Fix language key

Commit:
LPS-179282 Removes duplicate paths

Commit:
LPS-179282 Removes duplicate macros

Commit:
LPS-179282 Adjust the name of the macro

Commit:
LPS-179282 SF

gaborlovasdotcom committed:
LPS-173860 Temporary disable test case

nNilton committed:
LPS-184211 simplify

nNilton committed:
LPS-184211 add update tests

nNilton committed:
LPS-184211 sort

pedroliveirav committed:
LPS-184161 Fix global error in vite config

naraandrade committed:
LPS-178499 Fix update of currentRecipients

ling-alan-huang committed:
LRQA-80377 Auto SF

nNilton committed:
LPS-184211 rename ERC

manuferreira committed:
LRAC-13504 Add RequestToAccessDataForSpecificDocumentsAndMedia testcase

manuferreira committed:
LRAC-13504 Add RequestToAccessDataForAllDocumentsAndMedia testcase

manuferreira committed:
LRAC-1304 SF

Commit:
LPS-180075 Add CanUpdateCommerceProductGroupEntryAfterCreatingCommerceProductEntry test

Commit:
LPS-180079 Add CanUpdateOrderAfterUpdatingCommerceOrderEntry test

nNilton committed:
LPS-184211 make consistent

Commit:
LPS-181729 don't process invalid units

ling-alan-huang committed:
LPS-184438 Move to proper package

ling-alan-huang committed:
LPS-184438 Move to proper package

Commit:
LPS-181729 set default timeout to 30 seconds

joelrocha68 committed:
LPS-181578 Add setup and teardown

joelrocha68 committed:
LPS-181578 Add AnalystRoleCanNotCreateEditAndDeleteUser test

joelrocha68 committed:
LPS-181578 Add LeadRoleCanNotCreateEditAndDeleteUser test

joelrocha68 committed:
LPS-181578 Add UserRoleCanNotCreateEditAndDeleteUser test

joelrocha68 committed:
LPS-181578 Add NewUserWithAdmRoleCanCreateEditAndDeleteUser test

joelrocha68 committed:
LPS-181578 SF

ling-alan-huang committed:
LPS-184438 Move to proper package

joseabelenda committed:
LPS-184211 Simplify

Commit:
LPS-179282 Update deleteAllDatasetEntries macro

joseabelenda committed:
LPS-184211 Auto SF

Commit:
LRAC-13510 Add assertApiResponseNotContain macro

Commit:
LRAC-13510 Add RequestToAccessDataForSpecificChannelID testcase

joelrocha68 committed:
LPS-181578 Fix

manuferreira committed:
LRAC-13504 Add RequestToAccessDataForAllSegments testcase

manuferreira committed:
LRAC-13504 SF

thaynam committed:
LPS-184357 Fix Is possible to double claim the same activity

manuferreira committed:
LRAC-13504 Update testcase to create two segments and not rely on the demo data

manuferreira committed:
LRAC-13504 Add comment related to the teardown macro

Commit:
LPS-181729 avoid NPE

thaynam committed:
LPS-184476 Add activity record when MDF Request is in Draft state

Commit:
LPS-181729 add threshold to wait for delegate

Commit:
LPS-181729 buildLang

Commit:
LPS-181729 FS

Commit:
LPS-181729 widen the logCapture category

Commit:
LPS-181729 use companyId/userId defaults

Commit:
LPS-181729 add batch9 with missing entity

Commit:
LPS-181729 widen logcapture category

thaynam committed:
LPS-184476 Fix claim to account relationship name

thaynam committed:
LPS-184476 SF

gaborlovasdotcom committed:
LRQA-78829 Update XPath file to use new testing tool

gaborlovasdotcom committed:
LRQA-78829 Update macro file to use new testing tool

gaborlovasdotcom committed:
LRQA-78829 Update test cases to use new testing tool

Commit:
LPS-181729 add external reference code to the import task created from units

Commit:
LPS-181729 BatchEngineUnitProcessorTest

ling-alan-huang committed:
LPS-184495 Do not add modified method with @Modified annotation if it only calls deactivate and then activate

ling-alan-huang committed:
LPS-184495 Only enable for master for now

brianchandotcom committed:
LPS-184249 Wordsmith

brianchandotcom committed:
LPS-184249 Regen

brianchandotcom committed:
LPS-167608 Rename

brianchandotcom committed:
LPS-184003 Wordsmith

brianchandotcom committed:
LPS-183098 as used

brianchandotcom committed:
LPS-182073 Wordsmith

brianchandotcom committed:
LPS-182073 Regen

brianchandotcom committed:
LPS-174421, LPS-182413 Rename

brianchandotcom committed:
LPS-174421, LPS-182413 Rename

brianchandotcom committed:
LPS-174421, LPS-182413 As used

brianchandotcom committed:
LPS-174421, LPS-182413 auto SF

ling-alan-huang committed:
LPS-184495 Generated

ling-alan-huang committed:
LPS-175368 Need to check *.jsx for generating FeatureFlags

Commit:
LPS-184501 Increase timout explicit wait to make case stable

brianchandotcom committed:
LPS-183802 Wordsmith

brianchandotcom committed:
LPS-183802 Inline

brianchandotcom committed:
LPS-183802 prep next

brianchandotcom committed:
LPS-183802 Wordsmith

brianchandotcom committed:
LPS-183802 Regen

Commit:
LPS-184502 Modify macro to make it reusable

Commit:
LPS-184502 Add missing parameter

Commit:
LPS-184503 Modify case to adapt new behavior

brianchandotcom committed:
LPS-157718 Typo

Hanlf committed:
LPS-180322 Add and update tests about Document

Hanlf committed:
LPS-180322 Update macro to make it work for site document

Hanlf committed:
LPS-180322 Assert document is not added via API

gaborlovasdotcom committed:
LRQA-80627 Update XPath

bryceosterhaus committed:
LPS-183719 - Add lint rule and apply for TS interface naming

bryceosterhaus committed:
LPS-183719 - regen types

Commit:
LPS-184527 Update the number of adapted images

liferay-translation committed:
LPS-77699 Update Translations

joelrocha68 committed:
LPS-184529 Fix teardown

Commit:
LPS-181548 talend - simplify regex used for input field validation in Talend Open Studio (reported for ReDoS)

brianchandotcom committed:
saml-web 5.0.59 prep next

Commit:
LPS-181729 TEST WIP

Commit:
LPS-181729 manually update batch enabled ResourceImpls with VulcanBatchEngineTaskItemDelegate service

# Conflicts:
#	modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AdminAccountGroupResourceImpl.java
#	modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartItemResourceImpl.java

rotty3000 commented:
close, going to resend with my latest updates.
